ion-schema-tests-runner/src/generator.rs

```diff
@@ -123,6 +127,8 @@ fn generate_test_cases_for_file(ctx: Context) -> TokenStream {
         .read_all(&ion_content)
         .unwrap_or_else(|e| panic!("Error in {path_string} – {e:?}"));
 
+    let isl_version = find_isl_version(&schema_content);
```

Comment:
This file contains changes for ion-schema-tests-runner to consider version information when testing for invalid types.